scalableminds/webknossos
# Disable focus on tabbar in tracing view when tabs are selected

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the tracing view
- change the tab
- test the keyboard shortcuts; they should work without setting the focus back to the document

### Issues:
- fixes [#3039](https://github.com/scalableminds/webknossos/issues/3039)

------
